### PR TITLE
(cp*) bump v1.27.12

### PR DESCRIPTION
**Pull request by cbarria**
bump k8s version on all cp clusters to v1.27.12

### CODE REVIEW

**Review by jhoblitt — CHANGES_REQUESTED** [review]
The rancher.cp rancher `values.yaml` should be updated to remove the psp config (as it isn't relevant for k8s > 1.25).

The PR title also needs to be updated.

File: lukay/rke/cluster.yml
Comment (jhoblitt) [inline lukay/rke/cluster.yml:1]: The commit message for lukay is capitalized when it shouldn't be.

Comment (cbarria) [reply lukay/rke/cluster.yml:1]: fixed 👍🏻

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Pull request merged**